code/code/misc/periodic.cc

```diff
@@ -320,8 +320,9 @@ procSendGmcpTick::procSendGmcpTick(const int& p) {
 }
 
 void procSendGmcpTick::run(const TPulse&) const {
+  sstring gametime = GameTime::hmtAsString(GameTime::hourminTime());
   for (Descriptor* d = descriptor_list; d; d = d->next)
-    d->sendGmcp("comm.tick { }", false);
+    d->sendGmcp(format("comm.tick { \"time\": \"%s\" }") % gametime, false);
```

Comment:
So, part of the reason I started messing with the GMCP stuff is because I currently parse all these things using the old mudclient prompts - you can set them to send even if you're not using the official mudclient, and it gives you a bunch of extra data (but you have to parse it all out of the client codes and it gives you some messy updates).  We don't send nearly as much data via GMCP, which I thought was strange because the foundation was all there.   Anyway, MUD Time isn't something I would have thought to add, except we already send it via the mudclient prompts and it seemed like it would fit nicely in gmcp.comm.tick (which otherwise doesn't seem to do anything at all).  I keep it displayed using my client triggers, and I'm planning on modifying them to use GMCP instead of the mudclient prompts (so I can turn it off entirely, finally).